backup/src/main/java/io/camunda/zeebe/backup/api/BackupMetadata.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.backup.api;
+
+/** BackupMetadata must uniquely identify a backup stored in the BackupStore. */
+public interface BackupMetadata {
```

Comment:
:thought_balloon: I'm wondering if we can find a better name for this, something like `BackupIdentifier` (although that might be confused easily with `backupId`)

Reply:
@oleschoenburg One option is to name it `BackupIdentifier` and then rename the field `backupId` to `checkpointId`. wdyt? I can't think of a better name for the interface.

Reply:
Yeah, that sounds good to me :+1: